.github/workflows/explorer_index.yaml

```diff
@@ -0,0 +1,40 @@
+name: Explorer Index
+
+on:
+  push:
+    branches: [main]
+    paths:
+      - 'rules/**'
+
+jobs:
+  update-explorer-index:
+    runs-on: ubuntu-latest
+    
+    env:
+      AWS_S3_BUCKET: falco-distribution
+      AWS_S3_PREFIX: rules
+      AWS_S3_REGION: eu-west-1
+    
+    steps:
+      - name: Checkout Rules
+        uses: actions/checkout@v3
+
+      - name: Setup Golang
+        uses: actions/setup-go@v3
+        with:
+          go-version: '^1.20'
+
+      - name: Build registry artifact tool
+        working-directory: build/registry
+        run: cd rules_explorer && go run .
+      
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: "arn:aws:iam::292999226676:role/terraform-20230120142903096000000002"
+          aws-region: ${{ env.AWS_S3_REGION }}
+
+      - name: S3 Sync
+        uses: ItsKarma/aws-cli@v1.70.0
+        with:
+          args: s3 sync --delete --acl public-read rules_explorer/index.json s3://${{ env.AWS_S3_BUCKET }}/${{ env.AWS_S3_PREFIX }}/index.json
```

Comment:
Who is uploading the index.html file?

Reply:
It stays in Github, and we set Github Pages to display /rules_explorer. Only the index.json is stored in S3.

Reply:
Oh makes sense now, thanks!